envydis/gm107.c

```diff
@@ -231,7 +231,7 @@ static struct rbitfield u1334_bf = { { 34, 13 }, RBF_UNSIGNED };
 static struct rbitfield u1336_bf = { { 36, 13 }, RBF_UNSIGNED };
 static struct rbitfield u1620_bf = { { 20, 16 }, RBF_UNSIGNED };
 static struct rbitfield u1636_bf = { { 36, 16 }, RBF_UNSIGNED };
-static struct rbitfield u1920_bf = { { 20, 19 }, RBF_UNSIGNED };
+static struct rbitfield s2020_bf = { { 20, 19, 56, 1 }, RBF_SIGNED };
```

Comment:
Please maintain the current sort, both here and below. i.e. move this down with the signed ones.